contracts/interfaces/ISovryn.sol

```diff
@@ -111,6 +111,10 @@ contract ISovryn is
 
 	function setRebatePercent(uint256 rebatePercent) external;
 
+	function pause() external;
+
+	function unpause() external;
```

Comment:
why not just one function toggleProtocolPause() ?

Reply:
Removed Pause/Unpause and created a new function toggleFreeze(). Added test cases for one more module - LoanSettings